src/main/java/com/dguner/lombokbuilderhelper/LombokBuilderInspection.java

```diff
@@ -132,9 +132,12 @@ private boolean isClassBuilder(PsiClass aClass) {
     }
 
     private List<String> getMandatoryFields(PsiClass aClass) {
-        final Set<String> nonNullAnnotations =
-                Set.of("lombok.NonNull", "org.jetbrains.annotations.NotNull",
-                        "javax.validation.constraints.NotNull");
+        final Set<String> nonNullAnnotations = Set.of(
+            "lombok.NonNull",
+            "org.jetbrains.annotations.NotNull",
+            "javax.validation.constraints.NotNull",
+            "jakarta.validation.constraints.NotNull"
+        );
```

Comment:
https://blogs.oracle.com/javamagazine/post/transition-from-java-ee-to-jakarta-ee